js/src/LogEvent.js

```diff
@@ -51,7 +51,7 @@ class LogEvent
 
   nodeRequest(data, resolve, reject)
   {
-    let params = encodeURIComponent(JSON.stringify(data));
+    const { URL } = require('url');
```

Comment:
i wonder how compatible this is with browsers out there

Reply:
Are you about const { name } syntax? It is supported in all modern browsers (Chrome, Firefox, Safari, Edge). Also the above syntax is valid JS syntax, and since this function is never invoked on a browser side, then parsing of the file shouldn't be issue. I can double check in IE (and possibly in other non mainstream browsers?) if you are suspicious about this one.